From f81fe4c6f2a45d456c5481dbdd86d1d5cee4adb4 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Sun, 2 Dec 2018 10:07:22 +0200
Subject: [PATCH 0822/1051] arm64: dts: a8040-H: disable pcie1 on CP1

PCIE1 on CP1 was wrongly enabled.
Lane 4 on CP1 allocated for SFI 10G port.

Change-Id: Ia350dca11ec2d4216371a5b6570f11aba629b3a4
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/61446
Reviewed-by: Kostya Porotchkin <kostap@marvell.com>
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-8040-db-H.dts | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-H.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-H.dts
index 1caac6a006b7..44c86b7c0cc8 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-H.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-H.dts
@@ -95,12 +95,6 @@
 	phys = <&cp1_comphy0 0>;
 };
 
-/* CON7 on CP1 expansion */
-&cp1_pcie1 {
-	status = "okay";
-	phys = <&cp1_comphy4 1>;
-};
-
 /* CON5 on CP1 expansion */
 &cp1_pcie2 {
 	status = "okay";
-- 
2.17.1

